plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -127,6 +129,20 @@ public PluginManager(Config cfg) {
         skipFailedPlugins = cfg.isSkipFailedPlugins();
         hashFunction = cfg.getHashFunction();
         httpClient = null;
+        userAgentInformation = this.getUserAgentInformation();
+    }
+
+    private String getUserAgentInformation() {
+        String userAgentInformation= "Jenkins plugin manager/Dev";
```

Comment:
This value is invalid, no spaces are allowed in the user agent name. See https://tools.ietf.org/html/rfc7230#section-3.2.6 referenced from https://tools.ietf.org/html/rfc7231#section-5.5.3

Reply:
Should we replace them with hyphens then?

Reply:
Should we replace them with hyphens then?

Reply:
I submitted https://github.com/jenkinsci/plugin-installation-manager-tool/pull/324 with PascalCasing, I have no preference.

Reply:
Alright.